trikotFoundation/src/iosArm32Main/kotlin/com/mirego/trikot/foundation/concurrent/dispatchQueue/DispatchQueues.kt

```diff
@@ -0,0 +1,5 @@
+package com.mirego.trikot.foundation.concurrent.dispatchQueue
+
+actual class OperationDispatchQueue : iOSGlobalDispatchQueue(), DispatchQueue
+
+actual class SerialSubscriptionDispatchQueue : iOSSerialDispatchQueue("serialSubscriptionQueue"), DispatchQueue
```

Comment:
I think we could deprecate this one. The serialDispatchQueue was introduce as a way to deal with concurrency. A SerialQueue was added in the publishSubject to manage this concurrency.

Reply:
Added a deprecated annotation